app/views/shared/_profile-accordion-edit.html.erb

```diff
@@ -112,7 +115,35 @@
         <%= form.check_box :has_covid_symptoms, class: 'checkbox__input', checked: @contact.has_covid_symptoms == '1' %>
         <%= form.label :has_covid_symptoms, 'Are you or anyone in your home showing any symptoms of COVID-19?', class: 'checkbox__label' %>
       </div>
+
+    </fieldset>
+    
+  </div>
+
+  <button class="accordion__control" aria-expanded="true" id="contact-additional-info-accordion">
+    <h2 class="accordion__title">Consents</h2>
+  </button>
+  <div class="accordion__content">
+    <dl class="details-list">
+      <dt class="details-list__caption">Do not call</dt>
+      <dd class="details-list__value" id="no_calls_flag">
+        <%= friendly_boolean @contact.no_calls_flag %>
```

Comment:
Would we also want these to be editable here?

Reply:
Yes

Reply:
From Laura just now:
> when you click on add support action and it opens up the needs and the contact details to edit you should be ablet to add consent there
 but it currently doesn't appear
you can only add it by clicking edit their details
this isn't going to solve the current problem because users won't see it on calls